pygpudrive/env/config.py

```diff
@@ -22,15 +22,27 @@ class EnvConfig:
     road_obs_algorithm: str = "linear"
     obs_radius: float = 100.0
     polyline_reduction_threshold: float = 1.0
-
+    episode_length: int = 91
     # Action space (joint discrete)
     steer_actions: torch.Tensor = torch.round(
         torch.linspace(-1.0, 1.0, 13), decimals=3
     )
     accel_actions: torch.Tensor = torch.round(
         torch.linspace(-4.0, 4.0, 7), decimals=3
     )
-
+    dx: torch.Tensor = torch.round(
+        torch.linspace(-2.0, 2.0, 20), decimals=3
+    )
+    dy: torch.Tensor = torch.round(
+        torch.linspace(-2.0, 2.0, 20), decimals=3
+    )
+    dyaw: torch.Tensor = torch.round(
+        torch.linspace(-3.14, 3.14, 20), decimals=3
+    )
+    # Use bicycle model to infer expert actions
+    use_bicycle_model: bool = False
+    use_delta_model: bool = False
+    action_type: str = "bicycle"
```

Comment:
I'm a bit confused by the use of `action_type` here, it seems like it can be 'discrete', 'delta', or 'bicycle'

Would you mind briefly elaborating? It feels more natural to separate the dynamics_model from having a discrete/continuous action space

Reply:
Oh, I mean the action space of changing!! if we set as `bicycle`, action space set as `accel` and `steer`. On the other side, if we set as `delta`, action space will be `dx`, `dy`, and `dyaw`. It is not related with discrete/continuous space. Maybe think about other name will be better.

Reply:
But in this code, `action_typ` is not used, so i will delete it. (Now it defines in `env_torch.py`)